src/test/java/scenarios/Dashboard.java

```diff
@@ -0,0 +1,115 @@
+package scenarios;
+
+import org.automation.base.BaseTest;
+import org.automation.data.KadeUserAccount;
+import org.automation.pages.DashboardPage;
+import org.automation.session.KadeSession;
+import org.automation.utilities.Assertions;
+import org.automation.utilities.WebdriverWaits;
+import org.checkerframework.checker.units.qual.A;
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Dashboard extends BaseTest {
+
+    @Test(description=" DC01&2 Verify that following sections are displayed, on the 'Dashboard'")
+    public void verifyALLSectionsOnDashboardPage(){
+        KadeSession session = KadeSession.login(KadeUserAccount.Default);
+        DashboardPage Dashboard = new DashboardPage();
+        session.getSidePannel().expandManageBusinessAccordionBttn().click();
+        session.getSidePannel().getDashboardTab().click();
+
+        // verifying the sections on the Dashboard Page
+        //Your Business Section
+        Assertions.assertTrue(Dashboard.getYourBusinessTitle().isDisplayed());
+        String YourBusinessTitle= Dashboard.getYourBusinessTitle().getText();
```

Comment:
Avoid hardcoding strings like "Your Businesses". Can use constants or an external properties file to store static strings. This makes the code maintainable and reusable.